src/Stratis.Bitcoin.Features.MemoryPool/Fee/BlockPolicyEstimator.cs

```diff
@@ -23,55 +27,78 @@ namespace Stratis.Bitcoin.Features.MemoryPool.Fee
     /// at the bucket with the highest feerate transactions and verifying that a
     /// sufficiently high percentage of them were confirmed within 5 blocks and
     /// then you would look at the next highest feerate bucket, and so on, stopping at
-    /// the last bucket to pass the test.   The average feerate of transactions in this
+    /// the last bucket to pass the test.The average feerate of transactions in this
```

Comment:
One space could be useful there. Also please remove double spacing interest of the text. Also please use `<para>` to structure longer text.